website_sale_template_adj/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "eCommerce: View Adjustments",
+    "version": "12.0.1.0.1",
```

Comment:
I suggest "12.0.1.0.0" because it is still unpublished module. (just comment.)

Reply:
You're right!
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers